app/pods/publications/publication/documents/controller.js

```diff
@@ -229,152 +231,21 @@ export default class PublicationDocumentsController extends Controller {
-
-    // Visual stuff.
-    this.selectedPieces = [];
-
-    // Reset local activity to empty state.
-    this.previewActivity = {
-      mailContent: '',
-      mailSubject: '',
-      pieces: A([]),
-    };
-    this.showLoader = false;
-    this.renderPieces = true;
-    this.model.refreshAction();
-  }
-
-  /** TRANSLATION ACTIVITIES **/
-  @action
-  setTranslationMailSubject(event) {
-    set(this.translateActivity, 'mailSubject', event.target.value);
-  }
-
-  @action
-  async openTranslationRequestModal() {
-    this.translateActivity.finalTranslationDate = ((this.model.publicationFlow.translateBefore) ? this.model.publicationFlow.translateBefore : new Date());
-    this.translateActivity.pieces = this.selectedPieces;
-    const subject = await this.getConfig('email:translationRequest:subject', CONFIG.mail.translationRequest.subject);
-    const content = await this.getConfig('email:translationRequest:content', CONFIG.mail.translationRequest.content);
-    set(this.translateActivity, 'mailContent', await this.activityService.replaceTokens(content, this.model.publicationFlow, this.model.case));
-    set(this.translateActivity, 'mailSubject', await this.activityService.replaceTokens(subject, this.model.publicationFlow, this.model.case));
-    this.showTranslationModal = true;
-  }
-
-  get getTranslateActivityBeforeDate() {
-    if (this.model.publicationFlow.translateBefore) {
-      return this.model.publicationFlow.translateBefore;
-    }
-    return new Date();
-  }
-
   @action
-  async saveTranslationActivity() {
-    this.showLoader = true;
-    this.showTranslationModal = false;
-
-    // Fetch the type.
-    const translateSubCaseType = await this.store.findRecord('subcase-type', CONFIG.SUBCASE_TYPES.vertalen.id);
-
-    // TODO take from other subcase maybe?
-    const shortTitle = await this.model.case.shortTitle;
-    const title = await this.model.case.title;
-
-    // Find or Create subase.
-    const subcase = await this.subcasesService.findOrCreateSubcaseFromTypeInPublicationFlow(translateSubCaseType, this.model.publicationFlow, title, shortTitle);
-
-    // Create activity in subcase.
-    await this.activityService.createNewTranslationActivity(this.translateActivity.finalTranslationDate, this.translateActivity.mailContent, this.translateActivity.mailSubject, this.translateActivity.pieces, subcase);
-
-    // Send the email
-    this.emailService.sendEmail(CONFIG.EMAIL.DEFAULT_FROM, CONFIG.EMAIL.TO.translationsEmail, this.translateActivity.mailSubject, this.translateActivity.mailContent, this.translateActivity.pieces);
-
-    // Visual stuff.
-    this.selectedPieces = [];
-
-    // Reset local activity to empty state.
-    this.translateActivity = {
-      mailContent: '',
-      mailSubject: '',
-      finalTranslationDate: '',
-      pieces: A([]),
-    };
-    this.showLoader = false;
-    this.renderPieces = true;
-    this.model.refreshAction();
```

Comment:
Ik weet niet goed of ik dit mocht verwijderen. Misschien kon het gewoon in commentaar.